src/components/adminArea/Drawer/UserEditRoles.tsx

```diff
@@ -52,7 +63,7 @@ const UserEditRoles = ({ currentUser, role, setUserRole, shouldEdit }) => {
                         <fieldset
                             style={{
                                 padding: 8,
-                                border: `1px solid ${colors.bluePrimary}`,
+                                border: `1px solid ${getBorderColor(role)}`,
```

Comment:
chore (block): Instead passing the role as parameter, use the `nameSpace` variable, which is the value we use to determine platform color